input.cpp

```diff
@@ -12,80 +12,95 @@
 
 // functions that read from input files
 namespace marxan {
+    void to_lower(std::string& str)
+    {
+        transform(str.begin(), str.end(), str.begin(), ::tolower);
+    }
 
-    string storeFieldName(const vector<string>& varlist, int numvars, string sVarName,
-        const vector<string>& head, string fname) {
-
-        // add a field name from an input file header to a list
-        int foundit = 0;
-        int temp, newlink;
-
-        for (int i = 0; (i < numvars && foundit == 0); i++)
-        {
-            if (varlist[i].compare(sVarName) == 0)
-                foundit++;
-        }
-
-        if (!head.empty() && find(head.begin(), head.end(), sVarName) != head.end())
+    vector<string> get_tokens(const std::string& str)

```

Comment:
in marzone I removed the entire storeFieldName function as well and replaced with this: 

https://github.com/adelebai/marzone/blob/AlgorithmImprovements/util.hpp

See "getFileHeaders" function. I see you've rewritten a similar thing. Is there logic that we could consolidate?

Reply:
Sorry I commented in the wrong spot. Get_tokens looks reasonable and I see you've already removed storeFieldName. What I wanted to comment was actually removing GetFieldNames